### PR TITLE
SAD 2Q2016

### PR DESCRIPTION
**Pull request by pestOO**
Delivering Software Architecture Document in [DocDown](https://github.com/smartdevicelink/sdl_markdown_spec) format.
Converted to DocDown
Removed \<a\> links
Reordered for portal view
Updated `config.json` for testing on https://smartdevicelink.com/docs/
Added `README.md` with description and fast links to source documents
Added opensource and internal links to issues, question, requirements, specifications

Sources:
* [SAD as a confluence article](https://adc.luxoft.com/confluence/pages/viewpage.action?pageId=280330801)
* [auto-converted markdown SAD](https://github.com/smartdevicelink/sdl_core/blob/08be90e113d79af60d810d9f206f8e2f908c9e3b/docs/SAD.md)

Related issues: [APPLINK-26096](https://adc.luxoft.com/jira/browse/APPLINK-26096), [APPLINK-26666](https://adc.luxoft.com/jira/browse/APPLINK-26666)

### CODE REVIEW

**Comment by pestOO** [comment]
@justinjdickow, please look at this update and test on https://smartdevicelink.com/docs.

File: docs/Software Architecture Document/Table of Contects/index.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+## Table of contents
```
Comment (justinjdickow) [inline docs/Software Architecture Document/Table of Contects/index.md]: This file should be called "Table of Contents" not "Table of Contects"

**Comment by justinjdickow** [comment]
Preview this pull request at https://smartdevicelink.com/en/docs/pull_request/cd39ab920e12d6ae5a6f8fb424103f3da633eb68/overview/

File: docs/Software Architecture Document/Table of Contects/index.md (hunk 1 above)
Comment (pestOO) [inline docs/Software Architecture Document/Table of Contects/index.md]: @justinjdickow, fixup added.
it will be squashed after review.

**Comment by pestOO** [comment]
@justinjdickow, '404 error' for all sub-pages.
The most strange that `Configuration` and `TM Guide` are also not avalible, but in https://smartdevicelink.com/en/guides/core/overview/ working well.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+This is the open source SDL Core documentation:
+- [Software Architecture Document](docs/Software Architecture Document/Table of Contects/index.md)
```
Comment (justinjdickow) [inline README.md]: This should also be changed

**Comment by justinjdickow** [comment]
Preview this pull request at https://smartdevicelink.com/en/docs/pull_request/cd39ab920e12d6ae5a6f8fb424103f3da633eb68/overview/

**Comment by justinjdickow** [comment]
Preview this pull request at https://smartdevicelink.com/en/guides/pull_request/bc1f31465f91dbb9163224a209d8f45a72edede5/overview/

**Comment by justinjdickow** [comment]
Preview this pull request at https://smartdevicelink.com/en/guides/pull_request/787ed81b178d56fd624441ec9662dab85db1e0bd/overview/

**Comment by pestOO** [comment]
@justinjdickow,
GitHub MD format allows referring to headers.
It was used in ToC with references to sub-chapters as headers.
Is it possible to do with a DocDown right now?
If now, will it be okey to hold it for future usage?

**Comment by pestOO** [comment]
@justinjdickow,
This functionality with anchors or referring to header will be really good for providing direct links to SDL architecture and design.
[Example to component](https://github.com/pestOO/sdl_core_guides/blob/feature/SoftwareArchitectureDocument_2Q_baseline/docs/Software%20Architecture%20Document/Components%20View/index.md#resumption)
It could be also very helpful for guidelines.
[Example to digram](https://github.com/pestOO/sdl_core_guides/blob/feature/SoftwareArchitectureDocument_2Q_baseline/docs/Software%20Architecture%20Document/Deployment%20View/index.md#deployment-view-diagram)

**Comment by justinjdickow** [comment]
@pestOO Yes, you can use anchors to headers - here is an example: https://smartdevicelink.com/en/docs/hmi/master/common/structs/#buttoncapabilities

**Comment by justinjdickow** [comment]
Preview this pull request at https://smartdevicelink.com/en/guides/pull_request/17ab1265ad493e152972041d7a460c43f928bef3/overview/

**Comment by justinjdickow** [comment]
Preview this pull request at https://smartdevicelink.com/en/guides/pull_request/97c1bcf5b1c0cf74ae25f9c5ee63e38209ffc44f/overview/

**Comment by justinjdickow** [comment]
Preview this pull request at https://smartdevicelink.com/en/guides/pull_request/3c7ffbf5b068d015b2a8eb01eb029a3fe38ded89/overview/

**Comment by justinjdickow** [comment]
Preview this pull request at https://smartdevicelink.com/en/guides/pull_request/b30e5cd87ac38055feb6d3f6790a3339e65bf754/overview/

**Comment by pestOO** [comment]
@justinjdickow,
Solution diagram was updated and now mentioned as a 'Sequence diagram'.
Is it possible to change diagram type?

**Comment by justinjdickow** [comment]
Preview this pull request at https://smartdevicelink.com/en/guides/pull_request/c83efa7b863f39b904945c1045ea659cf7daeef9/overview/

**Comment by justinjdickow** [comment]
Preview this pull request at https://smartdevicelink.com/en/guides/pull_request/d5992a0a9a4490c1cfe7b1aa3e58a17d04436792/overview/

**Comment by justinjdickow** [comment]
Preview this pull request at https://smartdevicelink.com/en/guides/pull_request/4da329d828d6be1507cf757b7650ef5920c3869a/overview/

**Comment by justinjdickow** [comment]
Preview this pull request at https://smartdevicelink.com/en/guides/pull_request/49f536dab529e5efa69cee2ef868203031b656a2/overview/

**Comment by pestOO** [comment]
@justinjdickow,
Is it possible to use relative links to images in assets?
Using  `` shows empty picture.

**Pull request merged**